uint8 box_count
bbox_msg[] bbox_array
uint16[] bbox_ids